synapse/config/ratelimiting.py

```diff
@@ -46,6 +60,38 @@ def default_config(self, **kwargs):
         #
         rc_message_burst_count: 10.0
 
+        # Ratelimiting settings for registration.
+        rc_registration:
```

Comment:
can we make these be commented out in the generated config?

I'd recommend having a big comment block at the top explaining the structure, followed by a commented-out example. Something a bit like https://github.com/matrix-org/synapse/blob/b6ebef323e657c5413ac28d2dfb025b911e53cf1/docs/sample_config.yaml#L522.